### PR TITLE
Refactoring addImagesToProduct function

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Refactoring addImagesToProduct function
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | none
| Possible impacts? | none

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24288)

### CODE REVIEW

File: tests/UI/pages/BO/catalog/products/add.js [hunk 1]
```diff
@@ -95,6 +96,36 @@ class AddProduct extends BOBasePage {
     await page.keyboard.type(value);
   }
 
+  /**
+   * Get Number of images to set on the product
+   * @param page {Page} Browser tab
+   * @returns {Promise<number>}
+   */
+  async getNumberOfImages(page) {
+    return (await page.$$(this.imagePreviewBlock)).length;
```
Comment (Progi1984) [inline tests/UI/pages/BO/catalog/products/add.js:105]: Double dollar ?

**Review by Progi1984 — COMMENTED** [review]
A small question

File: tests/UI/pages/BO/catalog/products/add.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/pages/BO/catalog/products/add.js:105]: Yes, to get all elements matching the selector. Ref [here](https://playwright.dev/docs/api/class-page?_highlight=%24%24#pageselector-1)

**Review by Progi1984 — APPROVED** [review]

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by SD1982 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @boubkerbribri

**Pull request merged**